src/test/java/ru/training/at/hw4/testdata/TestData.java

```diff
@@ -0,0 +1,25 @@
+package ru.training.at.hw4.testdata;
+
+public class TestData {
+    public static final String homePageTitle = "Home Page";
+    public static final String userLoginName = "ROMAN IOVLEV";
+    public final static String homeHeader = "HOME";
+    public final static String contactFormHeader = "CONTACT FORM";
+    public final static String serviceHeader = "SERVICE";
+    public final static String metalsAndColorsHeader = "METALS & COLORS";
+    public final static String textFirstHomePage = "To include good practices\nand ideas from successful" +
+            "\nEPAM project";
+    public final static String textSecondHomePage = "To be flexible and\ncustomizable";
+    public final static String textThirdHomePage = "To be multiplatform";
+    public final static String textFourthHomePage = "Already have good base\n(about 20 internal and\n" +
+            "some external projects),\nwish to get more\u2026";
+    public final static String homeLeftMenu = "Home";
+    public final static String contactFormLeftMenu = "Contact form";
+    public final static String serviceLeftMenu = "Service";
+    public final static String metalAndColorsLeftMenu = "Metals & Colors";
+    public final static String elementsPackLeftMenu = "Elements packs";
+    public final static String logRowWater = "Water: condition changed to true";
+    public final static String logRowWind = "Wind: condition changed to true";
+    public final static String logRowSelen = "metal: value changed to Selen";
+    public final static String logRowYellow = "Colors: value changed to Yellow";
```

Comment:
`public static final` word order
Also constants should be named in **Snake Case (All Caps):** `USER_LOGIN_COUNT`